worker-message-prioritization-target-capacity-calculators/src/main/java/com/github/workerframework/workermessageprioritization/targetqueue/K8sTargetQueueSettingsProvider.java

```diff
@@ -105,29 +98,41 @@ private Map<String, TargetQueueSettings> getTargetQueueSettingsFromKubernetes()
         for (final String kubernetesNamespace : kubernetesNamespaces) {
 
             // Get all deployments in this namespace
-            final List<V1Deployment> deployments;
+            final List<IoK8sApiAppsV1Deployment> deployments;
             try {
-                deployments = Kubectl.get(V1Deployment.class).namespace(kubernetesNamespace).execute();
-            }  catch (final KubectlException kubectlException) {
+                deployments = appsV1Api.listAppsV1NamespacedDeployment(
```

Comment:
There may be a builder option that we could use when generating the client so that you don't have to pass all these nulls.

Reply:
I've added this option now:

```
<useSingleRequestParameter>true</useSingleRequestParameter>
```

https://github.com/CAFapi/caf-kubernetes-client/pull/1/files#diff-94ba2b5fecd028abe7c5887f44f34ca4dcd9437cd53676de10e4513ba13b81adR98